tests/test_dummy.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# MeshPy: A beam finite element input generator
+#
+# MIT License
+#
+# Copyright (c) 2018-2024
+#     Ivo Steinbrecher
+#     Institute for Mathematics and Computer-Based Simulation
+#     Universitaet der Bundeswehr Muenchen
+#     https://www.unibw.de/imcs-en
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+# -----------------------------------------------------------------------------
+"""Dummy test to demonstrate and ensure working pytest infrastructure."""
+
+from pathlib import Path
+
+import pytest
+
+
+def test_dummy(reference_file_directory: Path, tmp_path: Path, current_test_name: str):
+    """Dummy test to demonstrate pytest fixtures.
+
+    Args:
+        reference_file_directory: path to the reference file directory
+        tmp_path: temporary path for testing
+        current_test_name: name of the current test
+    """
+
+    # approach to get reference_file_directory
+    print("reference_file_directory: ", reference_file_directory)
+
+    # approach to get temporary testing path
+    print(
+        "tmp_path: ", tmp_path
+    )  # pytest automatically keeps the last three runs for debugging and deletes everything that's older
```

Comment:
Do you envision using this instead of the "testing-tmp" directory? In case CI tests fail we currently upload the `testing-tmp` directory to GitHub, to ease debugging. I would like to remain that possibility.

Reply:
I think the `tmp_path` approach is more viable as it is maintained through pytest and not our own implementation. 

For the CI we can also simply upload the test directory.

This also provides the benefit of even better management, e.g. see https://github.com/pytest-dev/pytest/issues/8141

We can then also only upload the data of the failed tests and trash the results of the successfull tests

Reply:
Sounds good!